activerecord/test/cases/invertible_migration_test.rb

```diff
@@ -96,7 +96,7 @@ class RemoveIndexMigration2 < SilentMigration
       def change
         change_table("horses") do |t|
           t.remove_index [:name, :color]
-          t.remove_index [:color], if_exists: true
+          t.remove_index [:color] if t.index_exists?(:color)
```

Comment:
Do we need to test both instead of changing the test? I think we also need a test that tests that the error is raised.